contracts/StakedCelo.sol

```diff
@@ -46,4 +67,72 @@ contract StakedCelo is ERC20Upgradeable, UUPSOwnableUpgradeable, Managed {
     function burn(address from, uint256 amount) external onlyManager {
         _burn(from, amount);
     }
+
+    /**
+     * @notice Locks stCELO from an address.
+     * @param account The address that will have its stCELO balance locked.
+     * @param amount The amount of stCELO to lock.
+     */
+    function lockBalance(address account, uint256 amount) external onlyManager {
```

Comment:
I think we should be careful with what we call "locked", because it means two different things:
(a) Celo in lockedGold contract and pending 3-day lockup and
b) that the Celo is voting and can't be unlocked before the a proposal passes/expires

i would suggest to differentiate the two, as I was quite confused while reading the code.

Reply:
changed to lockVoteStCelo, unlockVoteStCelo and so on